include/motor-control/core/motion_controller.hpp

```diff
@@ -79,7 +75,21 @@ class MotionController {
     MotionConstraints motion_constraints;
     GenericQueue& queue;
     CompletedQueue& completed_queue;
-    uint32_t steps_per_mm{0};
+    sq0_31 steps_per_mm{0};
+
+    Move convert_move_to_steps(
+        const can_messages::AddLinearMoveRequest& can_msg) {
```

Comment:
I don't understand what this type is.  It looks an anonymous function? Is that a thing in c++?

Reply:
I'm an idiot